index.html

```diff
@@ -5,47 +5,136 @@
+   </div>
     
-    <section id="About">
-      <h2>About</h2>
-        <p>I am a freelance web developer and designer specializing in high-quality Shopify & and WordPress development. I update, maintain, and troubleshoot websites.</p>
+    <section id="about">
+        <h2>About</h2>
+          <p>I am a freelance web developer specializing in high-quality Shopify & and WordPress development. I update, maintain, and troubleshoot websites.</p>
     </section> 
-
-    <section id = "Experience"> 
+    
+    
+    <section id = "experience"> 
       <h2>Experience</h2>
-      <ul>
-        <li>Data Analyst, Intern</li>
-        <li>Freelance Web Developer and SEO specialist</li>
-      </ul>
-      <p>Identified the clients and the project’s needs to increase their conversion rates and decrease their bounce rates.
+        <div class = "container">
+          <div>Data Analyst, Intern</div>
+          <div>Freelance Web Developer and SEO specialist</div>
+        </div>
+          <p class = "experience-section">Identified the clients and the project’s needs to increase their conversion rates and decrease their bounce rates.
         Stayed up-to-date on the latest SEO best practices.
         Developed on-site body content and page titles in support of SEO strategies.
         Optimized keyword use and effectiveness.
         Cross-tested website designs across multiple browsers.
         Managed WordPress content, designed and installed multiple plugins.
         Collaborated to manage all web change requests, updates including designing, developing new pages, sections as needed.
-        Built several responsive sites and completely from scratch made use of web tools to set up and manage websites using HTML5, CSS, JavaScript.</p>
+          </p>
+    </section> 
+      <h2>Take A Look At My Recent Projects</h2>
+    
+    <section id="projects" class="projects-section">
+        
+        <div class="project-card">
+            <img src="img/img_2.png" alt="Project 1" class="project-image">
+            <div class="project-details">
+                <h3>Project Title 1</h3>
+                <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
+                <a href="https://linktoproject1.com" target="_blank" class="project-link">View Project</a>
+            </div>
+        </div>
+        <div class="project-card">
+            <img src="img/img_2.png" alt="Project 2" class="project-image">
+            <div class="project-details">
+                <h3>Project Title 2</h3>
+                <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
+                <a href="https://linktoproject2.com" target="_blank" class="project-link">View Project</a>
+            </div>
+            
+        </div>
    
-      </section> 
+    </section>
+   
+    <section id="projects" class="projects-section">
+        
+      <div class="project-card">
+          <img src="img/img_2.png"" alt="Project 1" class="project-image">
```

Comment:
There's an extra `"` here.
```suggestion
          <img src="img/img_2.png" alt="Project 1" class="project-image">
```

Reply:
Updated